### ISSUE TITLE
The "Plugin URI" header in the plugin file is not valid... except it is

### ISSUE THREAD

**Issue by dartiss** [10up/plugin-check#771]
My plugin Code Embed (see https://wordpress.org/plugins/simple-embed-code/) has the following URI in the header...

```
 * Plugin URI:        https://wordpress.org/plugins/simple-embed-code/
 ```

However, plugin check is reporting it as invalid.

Code: plugin_header_invalid_plugin_uri_domain
Message: The "Plugin URI" header in the plugin file is not valid.

**Comment by swissspidy** [comment]
AFAIK the link should point to your website / the plugin‘s landing page, not just the plugin‘s listing in the plugin directory.

**Comment by dartiss** [comment]
The URL is valid though. If the real reason for the failure is because it's a wp.org domain, maybe it should specifically indicate that?

**Comment by ernilambar** [comment]
Yes, we could improve the message. Currently error is triggered for domain `wordpress.org` and `example.com`.

**Comment by ernilambar** [comment]
Message is coming from here https://github.com/WordPress/plugin-check/blob/trunk/includes/Checker/Checks/Plugin_Repo/Plugin_Header_Fields_Check.php#L116
Want to submit a PR?
